dev/com.ibm.ws.security.jwt/src/com/ibm/ws/security/jwt/internal/JwtConfigUtil.java

```diff
@@ -53,4 +58,27 @@ public static String processProtectedString(Map<String, Object> props, String cf
         return secret;
     }
 
+    public static String getSignatureAlgorithm(Map<String, Object> props, String sigAlgAttrName, String enableBetaAttrName) {
+        String defaultSignatureAlgorithm = "RS256";
+        String signatureAlgorithm = JwtUtils.trimIt((String) props.get(sigAlgAttrName));
+        boolean isBetaEnabled = (Boolean) props.get(enableBetaAttrName);
```

Comment:
Will this function work by default in the beta or will you have to set the config property even in the beta?